app/src/main/res/layout/fragment_profile.xml

```diff
@@ -38,16 +38,18 @@
                 />
 
             <TextView
+                android:id="@+id/tvName"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:text="Bio"
+                android:text="Name"
                 android:textSize="24sp"
                 />
+
             <TextView
-                android:id="@+id/tvBioContent"
+                android:id="@+id/tvUsername"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:text="Hey ho, willkommen auf meinem Profil 👋🏼"
+                android:text="username"
```

Comment:
Extract strings into resource file. However  I think in this case the android:text can be removed completely since the values get set when the data comes back. To check the layout with content rather use tools:text